charts/ratify/templates/configmap.yaml

```diff
@@ -42,7 +42,7 @@ data:
                 "name":"notaryv2",
                 "artifactTypes" : "application/vnd.cncf.notary.v2.signature",
                 "verificationCerts": [
-                    "/usr/local/ratify-certs/ratify-test.crt"
+                    "/usr/local/ratify-certs"
```

Comment:
Slightly unrelated to this PR but I found it a bit confusing at first whether `verificationCerts` wanted just the directory or the cert file path. Maybe we look into renaming the field in the future?

Reply:
ah, thanks for the feedback. We actually take both path ,and directory considering that some customer might specify a single certificate, and others might need to specify a directory if they have multiple.  Do we have any docs where we make explain each property? I should update that .

Reply:
Maybe somewhere [here](https://github.com/deislabs/ratify/blob/main/docs/verifier.md#example-verifier-configuration)? Although it's not exactly an explanation of the verifier config we just now.